src/procedures/IssueTokens.ts

```diff
@@ -19,16 +19,28 @@ import { ModifyShareholderData } from './ModifyShareholderData';
 import { Factories } from '../Context';
 import { ZERO_ADDRESS } from '../utils/constants';
 
+/**
+ * @hidden
+ */
 export const createRefreshSecurityTokenFactoryResolver = (
   factories: Factories,
   securityTokenId: string
 ) => async () => {
   return factories.securityTokenFactory.refresh(securityTokenId);
 };
 
+/**
+ * Procedure that issue new tokens and assigns them to the shareholders list
+ */
 export class IssueTokens extends Procedure<IssueTokensProcedureArgs, Shareholder[]> {
   public type = ProcedureType.IssueTokens;
 
+  /**
+   * - Issue a number of tokens and transfer to addresses of the investors list
+   *
+   * Note that this procedure will fail if the security token symbol doesn't exist
```

Comment:
Remove redundancy

No need for "symbol", upper case Security Token

Reply:
The second part wasn't addressed

Reply:
what do you mean?

Reply:
"upper case Security Token"

You missed that part